apps/sync/test/unit/crypto/stringconversion_test.js

```diff
@@ -85,6 +99,22 @@ suite('utils', () => {
           throw(Error);
     });
   });
+  suite('byteArrayToUtf16String', () => {
+    test('converts an Uint8Array to a UTF-16 string', () => {
+      var array = new Uint8Array(3);
+      array[0] = 0xe2;
+      array[1] = 0x82;
+      array[2] = 0xac;
```

Comment:
Those are the codes for UTF-8, the codes for UTF-16 are 0x20AC, see http://www.fileformat.info/info/unicode/char/20ac/index.htm